js-challenge/bbs-api/register.js

```diff
@@ -0,0 +1,57 @@
+'use strict';
+
+const p = console.log;
+const host = 'http://localhost'; //ローカルで掲示板を動かす場合の設定
+
+//以下にコードを書きましょう。
+const fetchForm = document.getElementById("fetchRegister");
+const btn = document.querySelector('.btn');
+
+const postRegister = () => {
+    const formData = new FormData(fetchForm);
+    fetch("http://localhost/register",
+        {
+            method: "POST",
+            body: formData
+        })
```

Comment:
インデントがずれてますね。vscodeで自動整形してください。prettierとかで。

```suggestion
    fetch("http://localhost/register",
        {
            method: "POST",
            body: formData
        }
    )
```